Product/Production/Gateway/AdminGUIWebservices/src/main/java/gov/hhs/fha/nhinc/admingui/loadtestdata/LoadTestDataManagement.java

```diff
@@ -0,0 +1,507 @@
+import gov.hhs.fha.nhinc.common.loadtestdatamanagement.ListAllPersonnameRequestMessageType;
+import gov.hhs.fha.nhinc.common.loadtestdatamanagement.ListAllPhonenumberRequestMessageType;
+import gov.hhs.fha.nhinc.common.loadtestdatamanagement.LoadTestDataSimpleResponseMessageType;
+import gov.hhs.fha.nhinc.common.loadtestdatamanagement.PatientType;
+import gov.hhs.fha.nhinc.common.loadtestdatamanagement.PersonnameType;
+import gov.hhs.fha.nhinc.common.loadtestdatamanagement.PhonenumberType;
+import gov.hhs.fha.nhinc.common.loadtestdatamanagement.SaveAddressRequestMessageType;
+import gov.hhs.fha.nhinc.common.loadtestdatamanagement.SaveDocumentRequestMessageType;
+import gov.hhs.fha.nhinc.common.loadtestdatamanagement.SaveEventCodeRequestMessageType;
+import gov.hhs.fha.nhinc.common.loadtestdatamanagement.SaveIdentifierRequestMessageType;
+import gov.hhs.fha.nhinc.common.loadtestdatamanagement.SavePatientRequestMessageType;
+import gov.hhs.fha.nhinc.common.loadtestdatamanagement.SavePersonnameRequestMessageType;
+import gov.hhs.fha.nhinc.common.loadtestdatamanagement.SavePhonenumberRequestMessageType;
+import gov.hhs.fha.nhinc.docrepository.adapter.model.DocumentMetadata;
+import gov.hhs.fha.nhinc.docrepository.adapter.model.EventCode;
+import gov.hhs.fha.nhinc.loadtestdata.LoadTestDataException;
+import gov.hhs.fha.nhinc.loadtestdata.LoadTestDataService;
+import gov.hhs.fha.nhinc.loadtestdatamanagement.EntityLoadTestDataManagementPortType;
+import gov.hhs.fha.nhinc.patientdb.model.Address;
+import gov.hhs.fha.nhinc.patientdb.model.Identifier;
+import gov.hhs.fha.nhinc.patientdb.model.Patient;
+import gov.hhs.fha.nhinc.patientdb.model.Personname;
+import gov.hhs.fha.nhinc.patientdb.model.Phonenumber;
+import org.apache.commons.lang.StringUtils;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.context.annotation.ImportResource;
+import org.springframework.stereotype.Component;
+
+/**
+ * @author ttang
+ *
+ */
+@ImportResource("file:${nhinc.properties.dir}/LoadTestDataConfig.xml")
+@Component
+public class LoadTestDataManagement implements EntityLoadTestDataManagementPortType {
+    private static final Logger LOG = LoggerFactory.getLogger(LoadTestDataManagement.class);
+    private static final String ACT_SUCCESSFUL = "successful";
+    private static final String ACT_FAIL = "fail";
+
+    @Autowired
+    private LoadTestDataService loadTestDataService;
+
+    @Override
+    public LoadTestDataSimpleResponseMessageType deleteAddress(DeleteAddressRequestMessageType request) {
+        Address record = new Address(request.getAddress(), loadTestDataService.getPatientBy(request.getAddress().getPatientId()));
+        if (null == record) {
+            return buildSimpleResponse(Boolean.FALSE, ACT_FAIL);
+        }
+        loadTestDataService.deleteAddress(record);
```

Comment:
Why are we assuming this passes? This method returns a boolean if it is successful or not - use that to build an ACT_SUCCESSFUL or ACT_FAIL message. As of right now its always a Success

This issue is repeated for most of these services